vhdbuilder/packer/components.json

```diff
@@ -131,8 +131,6 @@
     {
       "downloadURL": "mcr.microsoft.com/azuremonitor/containerinsights/ciprod/prometheus-collector/images:*",
       "amd64OnlyVersions": [
```

Comment:
at aks-rp side,  "5.3.0-main-10-06-2022-c0c49872" is used. you can double check with owner @vishiy

Reply:
Thanks, will ping him too. Double checked in aks-rp and only noticed the 5.3.0 version. 

Thanks Charlie, for sharing that this component was unused.